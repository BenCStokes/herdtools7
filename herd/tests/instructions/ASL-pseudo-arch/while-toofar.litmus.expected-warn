Warning: Loop unrolling reached limit, legal outcomes may be missing
