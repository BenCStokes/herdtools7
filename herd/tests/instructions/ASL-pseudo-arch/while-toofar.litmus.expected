Test while-toofar Required
States 3
0:T0.0.s=0;
0:T0.0.s=1;
0:T0.0.s=3;
Ok
Witnesses
Positive: 3 Negative: 0
Condition forall (true)
Observation while-toofar Always 3 0
Hash=86819245101076b2c307999d38623c84

